### PR TITLE
Prebuilt build script patches

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

File: prebuilt/scripts/ios.inc [hunk 1]
```diff
@@ -22,21 +22,47 @@ function addiOS {
 # Where to find Xcode installations
 XCODES_DIR="${HOME}/Builds/Platforms"
 
+ALT_XCODE_DIR=$(xcode-select -print-path)
+
```
Comment (peter-b) [inline prebuilt/scripts/ios.inc:40]: Try this:

```sh
function checkiOS() {
    local t_xcode_ver=$1
    local t_sdk_name=$2
    local t_sdk_ver=$3
    local t_archs=$4

    local t_try_xcode="${XCODES_DIR}/Xcode_${t_xcode_dir}"
    if [ -e "${t_try_xcode}" ]; then
        addiOS "${t_try_xcode}/Xcode.app/Contents/Developer" "${t_sdk_name}" "${t_sdk_ver}" "${archs}"
    elif [ -e "${ALT_XCODE_DIR}/Platforms/${t_sdk_name}.platform/Developer/SDKs/${t_sdk_name}${t_sdk_ver}.sdk" ]; then
        addiOS "${ALT_XCODE_DIR}" "${t_sdk_name}" "${t_sdk_ver}" "${archs}"
    fi
}

# Detect iOS variants
checkiOS 4_6_2 iPhoneSimulator 6.1 i386
checkiOS 5_1_1 iPhoneSimulator 7.1 i386
# etc...
```

Comment (montegoulding) [reply prebuilt/scripts/ios.inc:40]: that looks much nicer ;-)

Comment (peter-b) [reply prebuilt/scripts/ios.inc:40]: I haven't tested that it works, though.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 075499f

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 075499f2aae128dd274d0f6a7ab2d51d268927d5

**Comment by livecode-vulcan** [comment]
😎 test success 075499f2aae128dd274d0f6a7ab2d51d268927d5

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1045)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1048)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1040)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1046)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1050)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1062)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1040)

**Pull request merged**